res/poketool/pokegra/pl_pokegra/meson.build

```diff
@@ -0,0 +1,92 @@
+pl_pokegra_srcs = []
+pl_pokegra_files = []
+
+sprites_flags = ['-scanfronttoback', '-handleempty']
+palette_flags = ['-bitdepth', '8', '-nopad', '-comp', '10']
+
+foreach i : range(494)
+    padded_i = run_command('printf', '%04d', i.to_string(), check: false).stdout()
+
+    female_back = files(padded_i / 'female' / 'back.png')
+    male_back = files(padded_i / 'male' / 'back.png')
+    female_front = files(padded_i / 'female' / 'front.png')
+    male_front = files(padded_i / 'male' / 'front.png')
```

Comment:
Is explicit files() here really necessary? In many cases, these are automatically casted.